src/hooks/useUserLikes.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+import { User } from "firebase/auth";
+import { getUserLinksFromCollection } from "../firebase/userCollections";
+
+type UserLikes = {
+    url: string;
+    title: string;
+}
+
+const useUserLikes = (user: User) => {
+    const [initialLikes, setInitialLikes] = useState<UserLikes[] | null>(null);
+
+    useEffect(() => {
+        async function getUserLikes () {
+            const likes = await getUserLinksFromCollection('Likes', user) as UserLikes[];
+            setInitialLikes(likes);
+        };
+
+        if (user) {
+            setInitialLikes(null);
+            getUserLikes();
+        };
+    }, [user]);
+
+    return { initialLikes };
+}
+
+export default useUserLikes;
```

Comment:
As commented above, please use Firebase methods to update `likes` automaticaly here and revert changes related to GlobalContext.

Reply:
Do you mean, use oneSnapShot from firestore with GlobalContext just like the "Stats" method ?

Reply:
Especially this comment: https://github.com/agenciagift/feed/pull/7/files#r1178584482

Reply:
i can't find these comments if you could post here a print or something please.

Reply:
Let me know if you can't see this:

https://user-images.githubusercontent.com/1944417/235928410-10ed2452-9b02-4459-829b-6307dd51419d.mov

Reply:
Thanks for the vid really helped!